<?xml version="1.0"?>

<launch>

	<!-- Run move base -->
	<node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen" >

		<param name="GlobalPlanner/use_dijkstra" value="true" />
		<param name="base_local_planner" value="teb_local_planner/TebLocalPlannerROS" />
		
		<rosparam file="$(find lns_navigation)/params/sim/costmap_common_params.yaml" command="load" ns="global_costmap" />
		<rosparam file="$(find lns_navigation)/params/sim/costmap_common_params.yaml" command="load" ns="local_costmap" />

		<rosparam file="$(find lns_navigation)/params/sim/local_costmap_params.yaml" command="load" />
		<rosparam file="$(find lns_navigation)/params/sim/global_costmap_params.yaml" command="load" />

		<rosparam file="$(find lns_navigation)/params/sim/teb_planner_params.yaml" command="load" />
		<rosparam file="$(find lns_navigation)/params/sim/move_base_params.yaml" command="load" />

		<remap from="cmd_vel" to="/four_wheel_steering_controller/cmd_vel" />


	</node>

	<node name="rviz" pkg="rviz" type="rviz" args="-d $(find lns_navigation)/rviz/nav.rviz" required="true" />


</launch>